vividus-extension-selenium/src/main/java/org/vividus/ui/ContextSourceCodeProvider.java

```diff
@@ -23,4 +23,6 @@ public interface ContextSourceCodeProvider
     String APPLICATION_SOURCE_CODE = "Application source code";
 
     Map<String, String> getSourceCode();
+
+    Map<String, String> getShadowDomSourceCode();
```

Comment:
it is applicable for web only, thus we should introduce it in `vividus-extension-web-app`